### PR TITLE
add more descriptive plugin docs

### PR DESCRIPTION
**Pull request by talevy**

### CODE REVIEW

File: lib/logstash/codecs/avro.rb [hunk 1]
```diff
@@ -6,6 +6,46 @@
 require "logstash/timestamp"
 require "logstash/util"
 
+# Read serialized Avro records as Logstash events
+#
+# This plugin is used to serialize Logstash events as 
+# Avro datums, as well as deserializing Avro datums into 
+# Logstash events.
+#
```
Comment (suyograo) [inline lib/logstash/codecs/avro.rb]: Can you add something like "Example usage is with Kafka input/output"

**Comment by suyograo** [comment]
Minor comment, LGTM

File: lib/logstash/codecs/avro.rb (hunk 1 above)
Comment (talevy) [inline lib/logstash/codecs/avro.rb]: should I just rename the `=== Usage` below?

Comment (talevy) [inline lib/logstash/codecs/avro.rb]: updated: explicitly included this description within the usage section below

Comment (suyograo) [inline lib/logstash/codecs/avro.rb]: +1

**Comment by suyograo** [comment]
LGTM

**Pull request closed without merging**